airbyte-webapp/src/views/layout/SideBar/components/MenuContent.tsx

```diff
@@ -0,0 +1,19 @@
+import { PropsWithChildren } from "react";
+
+import { FlexContainer } from "components/ui/Flex";
+
+import styles from "./MenuContent.module.scss";
+
+export const MenuContent: React.FC<PropsWithChildren<unknown>> = ({ children }) => {
```

Comment:
essentially just a wrapper for a full-height full-width flex container so I wasn't constantly overriding in separate files with the same className.  

Does make me wonder whether we want that as a prop on the `FlexContainer` itself... but _really_ trying to watch scope creep at this point as this PR is getting gnarly.

Reply:
It does seem like a common use case, however when I looked through our code I only saw 2 places we were sending a classname with a width or height of 100% via a `className` prop... so I am on the fence...